src/components/beat.js

```diff
@@ -299,7 +290,7 @@ AFRAME.registerComponent('beat', {
     this.rigContainer = document.getElementById('rigContainer');
     this.superCuts = document.querySelectorAll('.superCutFx');
 
-    this.verticalPositions = this.beatSystem.verticalPositions;
```

Comment:
need to rename variable? I would keep it as it was. Review is easier

Reply:
It was a bit easier to avoid making sure I didn't accidentally use one in place of the other, since the contents and interpretation of the field is different. But yes I can change the name back.